generators/app/templates/src/contexts/profile-shape.json

```diff
@@ -0,0 +1,48 @@
+{
+  "profile": {
```

Comment:
I don't think this is good JSON-LD (doesn't seem to make any sense with https://json-ld.org/playground/ at least); I would prefer to have valid JSON-LD if we're using the @context property.

Reply:
Indeed, bring the `@context` one level higher.

Reply:
sure I will fix it, thank you !!!

Reply:
{
   "@context": {},
  " profile": [{}],
}

Do you like this one ? I also removed shape key so now profile is an array

Reply:
Fine with me.